caveclient/chunkedgraph.py

```diff
@@ -1033,9 +1047,9 @@ def get_latest_roots(
             logger.warning("timestamp_future is deprecated, use timestamp instead")
             timestamp = timestamp_future
 
-        if timestamp is None:
-            timestamp = datetime.datetime.now(datetime.timezone.utc)
-        elif timestamp.tzinfo is None:
+        timestamp = self._process_timestamp(timestamp)
```

Comment:
This is a slight change in logic. This code looks like it was not using the ._default_timestamp of the client previously